packages/venia-concept/src/RootComponents/Category/category.css

```diff
@@ -18,3 +18,11 @@
 .placeholder {
     height: 100vh;
 }
+
+.categoryTitle {
+    color: black;
```

Comment:
`black` should be `var(--venia-text)`

Reply:
I don't know how all these GitHub review tools work, so I guess I'll just... make another PR... and put the link here?

Reply:
PR: https://github.com/magento-research/pwa-studio/pull/808